w3d_translator/read_xml.py

```diff
@@ -1,13 +1,26 @@
 import xml.etree.ElementTree as ET
+from pathlib import Path
 
-from classes import Color, WandNavigation
+from utils import str_to_tuple, tf_to_bool
 
 
 def read_xml(file):
     xml = ET.parse(file)
     root = xml.getroot()
+    story = {
+        "version": int(root.attrib["version"]),
+        "last_xpath": Path(root.attrib["last_xpath"]),
+    }
 
-    story = {}  # TODO: Make story its own class (5)
+    # Globals
+    g = root.find("Global")
+    story["Camera"] = parse_camera(g.find("CameraPos"))
+    story["CaveCamera"] = parse_camera(g.find("CaveCameraPos"))
+    story["background"] = str_to_tuple(g.find("Background").attrib["color"])
+    story["wand_navigation"] = parse_wand_navigation(g.find("WandNavigation"))
+
+    story["walls"] = parse_placements(root.find("PlacementRoot"))
```

Comment:
Yeah you may be right. I want to differentiate the `<Placement>`'s in `<PlacementRoot` from those in the `Camera`, `Object`, etc tags. The "walls" are really reference points and will end up becoming GameObjects themselves in Unity whereas a Placement in the other roots are really just a relative position.

Perhaps I use `story["placements"]` instead of `story["walls"]` but rename the tag when I find it elsewhere? Struggling to come up with a pertinent variable name though